### PR TITLE
Document RedirectResponse::$url

### PR DESCRIPTION
**Pull request by dawehner**
| Q | A |
| --- | --- |
| Bug fix? | No |
| New feature? | No |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets |  |
| License | MIT |
| Doc PR |  |

### CODE REVIEW

File: src/Symfony/Component/HttpFoundation/RedirectResponse.php [hunk 1]
```diff
@@ -23,9 +23,10 @@ class RedirectResponse extends Response
     /**
      * Creates a redirect response so that it conforms to the rules defined for a redirect status code.
      *
-     * @param string $url     The URL to redirect to
-     * @param int    $status  The status code (302 by default)
-     * @param array  $headers The headers (Location is always set to the given URL)
+     * @param string  $url     The URL to redirect to. The URL should be a full URL, with schema etc.,
+     *                         but pratically every browser redirects on paths only as well.
```
Comment (iltar) [inline src/Symfony/Component/HttpFoundation/RedirectResponse.php]: What about something as simple as "The absolute URL or relative path to redirect to"?

Comment (Tobion) [inline src/Symfony/Component/HttpFoundation/RedirectResponse.php]: a relative path is not the same as "path only" as he described. but I guess what browsers support is URL or URL reference.

File: src/Symfony/Component/DomCrawler/Tests/Field/TextareaFormFieldTest.php [hunk 2]
```diff
@@ -43,4 +43,5 @@ public function testInitialize()
 
         $this->assertEquals('foo bar <h1>Baz</h2>', $field->getValue(), '->initialize() sets the value of the field to the textarea node value');
     }
+
```
Comment (fabpot) [inline src/Symfony/Component/DomCrawler/Tests/Field/TextareaFormFieldTest.php]: should be reverted

**Comment by fabpot** [comment]
@dawehner Can you rebase to get rid of the merge commit?

File: src/Symfony/Component/HttpFoundation/RedirectResponse.php [hunk 3]
```diff
@@ -23,9 +23,10 @@ class RedirectResponse extends Response
...
+     * @param integer $status  The status code (302 by default)
```
Comment (fabpot) [inline src/Symfony/Component/HttpFoundation/RedirectResponse.php]: should be kept as `int`

**Comment by fabpot** [comment]
59944896f28c11bf9d94e03b6f04cea33426eda8

**Comment by dawehner** [comment]
Thank you @fabpot  and sorry for slacking

**Pull request closed without merging**